packages/utils/src/errors.js

```
@@ -200,6 +200,13 @@ export class Dhis2Error extends RespondingError {
   }
 }
 
+export class UnsupportedMediaTypeError extends RespondingError {
```

The current usage of this actually gets swallowed and rethrown as a `500 Internal Server Error`. Not fixing that for now, but I’d like this error to be ready anyway